File: angular-app/src/app/features/blog/class/Blog.ts
export interface BlogInterface {
    title: string;
    image: string;
    description: string;
    cols: number;
    rows: number;
}

export class Blog {
    private _title: string;
    private _image: string;
    private _description: string;
    private _cols: number;
    private _rows: number;

    constructor(title: string, image: string, description: string) {
        this._title = title;
        this._image = image;
        this._description = description;
        this._cols = 1;
        this._rows = 2;
    }

    /**
     * Getter title
     */
    public get title(): string {
        return this._title;
    }

    /**
     * Setter title
     */
    public set title(value: string) {
        this._title = value;
    }

    /**
     * Getter image
     */
    public get image(): string {
        return this._image;
    }

    /**
     * Setter image
     */
    public set image(value: string) {
        this._image = value;
    }

    /**
     * Getter description
     */
    public get description(): string {
        return this._description;
    }

    /**
     * Setter description
     */
    public set description(value: string) {
        this._description = value;
    }

    /**
     * Getter cols
     */
    public get cols(): number {
        return this._cols;
    }

    /**
     * Setter cols
     */
    public set cols(value: number) {
        this._cols = value;
    }

    /**
     * Getter rows
     */
    public get rows(): number {
        return this._rows;
    }

    /**
     * Setter rows
     */
    public set rows(value: number) {
        this._rows = value;
    }
}

File: blog-service/src/main/java/com/springsetup/forumservice/model/Blog.java
package com.springsetup.forumservice.model;

import javax.persistence.*;

@Entity
@Table(name="blog")
public class Blog {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    @Column(name="title")
    private String title;
    @Column(name="image")
    private String image;
    @Column(name="description")
    private String description;
    @Column(name="cols_ui")
    private int cols;
    @Column(name="rows_ui")
    private int rows;

    public Blog() {
    }

    public Blog(String title, String image, String description, int cols, int rows) {
        this.title = title;
        this.image = image;
        this.description = description;
        this.cols = cols;
        this.rows = rows;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getCols() {
        return cols;
    }

    public void setCols(int cols) {
        this.cols = cols;
    }

    public int getRows() {
        return rows;
    }

    public void setRows(int rows) {
        this.rows = rows;
    }

    @Override
    public String toString() {
        return "Blog{" +
                "id=" + id +
                ", title='" + title + '\'' +
                ", image='" + image + '\'' +
                ", description='" + description + '\'' +
                ", cols=" + cols +
                ", rows=" + rows +
                '}';
    }
}
